pymongo/database.py

```diff
@@ -895,7 +895,17 @@ def current_op(self, include_all=False, session=None):
         return self._current_op(include_all, session)
 
     def profiling_level(self, session=None):
-        """Get the database's current profiling level.
+        """**DEPRECATED**: Get the database's current profiling level.
+
+        Starting with PyMongo 3.12, this helper is obsolete. The functionality
+        provided by this helper is available in MongoDB 3.6+ using the
+        `profile command`_, which can be run using the :meth:`command`
+        helper. Running the `profile command`_ with the level set to ``-1``
+        returns the current profiler info without changing it::
+
+           profile_info = db.command({'profile': -1})
```

Comment:
Suggest renaming profile_info to `res` or `status` to avoid naming confusion with the "profile_info()" method. It would also be helpful to add some parsing of the response:
```
res = db.command('profile', -1)
profiling_level = res['was']
slowms = res['slowms']